app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/SalutationHelper.scala

```diff
@@ -30,7 +32,10 @@ object SalutationHelper {
       }.mkString
     }
 
-  def salutationFrom(params: Map[String, Any]): String = {
+  def salutationFrom(params: Map[String, Any], salutationText: String = EnglishSalutation): String = {
```

Comment:
It is not clear what is the purpose of the new argument `salutationText`. I didn't find any single usage.

Reply:
Thank you I will change it